typegate/src/runtimes/prisma/hooks/run_migrations.ts

```diff
@@ -0,0 +1,153 @@
+// Copyright Metatype OÜ, licensed under the Elastic License 2.0.
+// SPDX-License-Identifier: Elastic-2.0
+
+import { PushHandler } from "../../../typegate/hooks.ts";
+import { makeDatasource } from "../prisma.ts";
+import { PrismaRT } from "../mod.ts";
+import * as native from "native";
+import { nativeResult, pluralSuffix } from "../../../utils.ts";
+
+export const runMigrations: PushHandler = async (
```

Comment:
should not hook by declared close the runtime class as they will likely related to them? this will give an easier structure for later stage plugin?